### DISCUSSION TITLE
Web pages are rendered differently on PC than on mobile

### DISCUSSION THREAD

**Discussion by People-11** [spencerwooo/onedrive-vercel-index#542] [Q&A] [answered]
One example：
On PC, the elements on NavBar will near the left and right edges due to justify-between attributes
![image_2022-03-15_19-36-40](https://user-images.githubusercontent.com/41558215/158370755-cee0439f-72b0-4ad7-8a62-15bd6cdd552e.png)
But on phone, if I delete Search Bar (due to terrible search experience I rather remove it), the elements will close to the left, seems not obeying the justify-between attribute.
![Screenshot_20220315-194433_Chrome~2](https://user-images.githubusercontent.com/41558215/158371120-ee13afcd-9042-40f7-9921-6981b932261a.png)

Another example:
On the audio playback page, I deleted m-4 attribute to remove the grey square to make the picture appear bigger. 
![IMG_20220315_195345_463](https://user-images.githubusercontent.com/41558215/158372491-febac9f7-996f-4d42-923f-9ea367e156f2.jpg)
But on phone, the grey square still appeared, which make the picture appears small.
![Screenshot_20220315-195446_Chrome~2](https://user-images.githubusercontent.com/41558215/158372683-2c1780cd-348c-445e-9b3a-ec0b383541dd.png)

So is there anyway to make the appearance on phone in line with my modified configuration? Or is there any special configuration that force the phone web interface render like that?

**Comment by spencerwooo** [comment, marked as answer]
You may be looking for [tailwind responsive design](https://tailwindcss.com/docs/responsive-design).
